src/Results/Results.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './Results.scss'
+
+interface ResultsProps {
+	searchResults: Array<any>
```

Comment:
Cool, calling functions looks much easier than I thought they would be with TS. Great work!

Reply:
^^^